math/src/field/fields/montgomery_backed_prime_fields.rs

```diff
@@ -208,6 +210,33 @@ where
     }
 }
 
+impl<M, const NUM_LIMBS: usize>
+    Distribution<FieldElement<MontgomeryBackendPrimeField<M, NUM_LIMBS>>> for Standard
+where
+    M: IsModulus<UnsignedInteger<NUM_LIMBS>> + Clone + Debug,
+{
+    fn sample<R: Rng + ?Sized>(
+        &self,
+        rng: &mut R,
+    ) -> FieldElement<MontgomeryBackendPrimeField<M, NUM_LIMBS>> {
+        let mut rand_limbs: [u64; NUM_LIMBS] = [0; NUM_LIMBS];
+        let mut roll_to_max: bool = false;
+        let mod_min_one = M::MODULUS - UnsignedInteger::from_u64(1_u64);
+
+        #[allow(clippy::all)] //Lint imposed by clippy makes code harder to read.
```

Comment:
`clippy::all` seems excessive. Try to limit it to the actual lint that causes problems.